compiler/parser/src/tokenizer/lexer.rs

```diff
@@ -426,6 +426,7 @@ impl Token {
                     "scalar" => Token::Scalar,
                     "self" => Token::SelfLower,
                     "string" => Token::String,
+                    "struct" => Token::Struct,
```

Comment:
Should we also remove `circuit` from this list of keywords, since it should be now available as an identifier?